packages/app-security-access-management/src/ui/views/Groups/GroupsForm.tsx

```diff
@@ -155,6 +156,26 @@ export const GroupsForm = () => {
                         {loading && <CircularProgress />}
                         <SimpleFormHeader title={data.name ? data.name : "Untitled"} />
                         <SimpleFormContent>
+                            {systemGroup && (
+                                <Grid>
+                                    <Cell span={12}>
+                                        <Alert type={"info"} title={"Permissions are locked"}>
+                                            This is a protected system role and you can&apos;t
+                                            modify its permissions.
+                                        </Alert>
+                                    </Cell>
+                                </Grid>
+                            )}
+                            {pluginGroup && (
+                                <Grid>
+                                    <Cell span={12}>
+                                        <Alert type={"info"} title={"Permissions are locked"}>
+                                            This role is registered via an extension. It cannot be
+                                            modified via the Admin Area.
```

Comment:
Suggesting a simpler alternative:
> This role is registered via an extension, and cannot be modified.

It's shorter, and I would really like to avoid this `Admin Area`, since we don't have a standard for this even in our docs. Hurts less by simply avoiding that in the first place.

Reply:
Done.